ivy/functional/backends/tensorflow/experimental/creation.py

```diff
@@ -158,3 +158,22 @@ def mel_weight_matrix(
         lower_edge_hertz=lower_edge_hertz,
         upper_edge_hertz=upper_edge_hertz,
     )
+
+
+def unsorted_segment_mean(
+    data: tf.Tensor,
+    segment_ids: tf.Tensor,
+    num_segments: Union[int, tf.Tensor],
+) -> tf.Tensor:
+    # Calculate the sum along segments
+    segment_sum = tf.math.unsorted_segment_sum(data, segment_ids, num_segments)
```

Comment:
tf seems to have [this](https://www.tensorflow.org/api_docs/python/tf/math/unsorted_segment_mean) already, any specific reason to reimplement i might be missing? 🤔

Reply:
Done. Used TensorFlow mean function.